### PR TITLE
Add specs book links and removed language/rest specs

### PR DESCRIPTION
**Pull request by jewzaam**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,10 +1,13 @@
-[![Build Status](https://travis-ci.org/lightblue-platform/lightblue-core.svg?branch=master)](https://travis-ci.org/lightblue-platform/lightblue-core) [![Coverage Status](https://coveralls.io/repos/lightblue-platform/lightblue-core/badge.png?branch=master)](https://coveralls.io/r/lightblue-platform/lightblue-core?branch=master) [![Stories in Ready](https://badge.waffle.io/lightblue-platform/lightblue.svg?label=ready&title=Ready)](http://waffle.io/lightblue-platform/lightblue)
```
Comment (luan-cestari) [inline README.md]: Nice, I noticed that you referenced the lightblue parent projects (that have the other ones as maven modules) as it has the overall coverage metrics. 

Another point that I saw it it used to reference our waffle account, do you think it doesn't have to be referenced there anymore? I thought that link could be helpful so see a new feature on roadmap or maybe a bug.

**Comment by luan-cestari** [comment]
@jewzaam , commented a single line. Just let me know what you think about that comment and then I will merge (everything else seem fine)

Just to mention in this PR, the new repository with the moved content is the https://github.com/lightblue-platform/lightblue-docs-specs  and you can see it online on the link http://jewzaam.gitbooks.io/lightblue-specifications/content/ (might change this link to use the lightblue's cname)

**Comment by luan-cestari** [comment]
Another reference http://dev.forum.lightblue.io/New-quot-specifications-quot-gitbook-td350.html

File: README.md (hunk 1 above)
Comment (jewzaam) [inline README.md]: The waffle batch only shows issues in a "ready" status.  And I think it picks only for the primary repository, which is https://github.com/lightblue-platform/lightblue.  That means it always shows "Ready: 0". So, there is no value.  See http://docs.lightblue.io/

**Comment by jewzaam** [comment]
CNAME request for specs.docs.lightblue.io is submitted as of last week, gitbook config will be updated when it's available.

File: README.md (hunk 1 above)
Comment (luan-cestari) [inline README.md]: Sorry, let me put this other way. I think that we should put a reference to waffle somewhere. I didn't find any link for that and I think it is important (maybe we could put on the overview repo) , what do you think?. Sorry about the misunderstanding

**Comment by luan-cestari** [comment]
OK. 

I just updated the commented line thread.

**Comment by jewzaam** [comment]
We have  waffle documented in the contribution process: http://dev.docs.lightblue.io/development_process/README.html

**Comment by luan-cestari** [comment]
OK, fair enough. I will merge it now

**Pull request merged**